package.json

```diff
@@ -87,8 +91,11 @@
     "loader.js": "^4.2.3",
     "mktemp": "^0.4.0",
     "mocha": "^5.0.0",
+    "rimraf": "^2.6.2",
     "testdouble": "^3.5.0",
-    "typescript": "^2.7.2"
+    "ts-node": "^7.0.1",
+    "typescript": "^2.7.2",
```

Comment:
Suggestion for follow-up: given this is for *us*, let's update soon to use latest TS here. It fixes quite a few bugs in the type system from what I've seen, and while we're unlikely to hit most of them outside Ember code.